hibike/devices/LimitSwitch/limit_switch.h

```diff
@@ -21,5 +21,6 @@ hibike_uid_t UID = {
 void setup();
```

Comment:
I believe that Twistit supports 3 switches, not 4.  A0 through A2.  Please updated NUM_SWITCHES.

Also, please updated The year #